src/platform/packages/private/kbn-scout-reporting/src/reporting/report/events/event.ts

```diff
@@ -85,6 +85,7 @@ export interface ScoutTestRunInfo {
 export interface ScoutSuiteInfo {
   title: string;
   type: string;
+  file?: ScoutFileInfo;
```

Comment:
Please remove this and use `ScoutTestInfo['file']` or `ScoutTestRunInfo['config']['file']` instead.